### ISSUE TITLE
SyntaxError: Unexpected token 'export'

### ISSUE THREAD

**Issue by catrawalkar** [Swizec/useAuth#135]
When I run next-js project with simple setup, I get this error.

```
error - /home/chinmay/Projects/next-useauth-starter/node_modules/react-use-auth/auth0.js:4
export { Auth0 } from "./dist/auth0";
^^^^^^

SyntaxError: Unexpected token 'export'
```

My code:
__app.js
```
return (
    <>
      <AuthConfig
        navigate={(url) => router.push(url)}
        authProvider={Auth0}
        params={{
          domain: "useauth.auth0.com",
          clientID: "GjWNFNOHqlino7lQNjBwEywalaYtbIzh",
        }}
      />

      <Component {...pageProps} />
    </>
  );
```

**Comment by catrawalkar** [comment]
@jasonLaster can you confirm if you are facing this too?

**Comment by Swizec** [comment]
Can you share your repo?
What version of Node are you on?

**Comment by jasonLaster** [comment]
node v12.16.3. Looks like it also repros with `v14.15.3`

**Comment by d-sanderson** [comment]
also experiencing the same error when upgrading from `pre-1.0.0` here is a link to a reproduction [repo](https://github.com/d-sanderson/useauth-create-next-app- ) running node `14.2.0`

**Comment by benheb** [comment]
Same issue running node `v12.19.0`. Any updates here?

**Comment by benheb** [comment]
This seems to be a workaround: `import { Auth0 } from 'react-use-auth/dist/auth0'`

**Comment by Swizec** [comment]
Thanks to @jasonLaster for the fix ❤️ 

Just published v2.0.3, you should be good now @catrawalkar @d-sanderson @benheb. Closing for now

**Comment by arronhunt** [comment]
I also ran into this issue with Netlify Identity. I used @benheb's solution to get it to work

```jsx
import { NetlifyIdentity } from "react-use-auth/netlify-identity";
// Change to:
import { NetlifyIdentity } from "react-use-auth/dist/NetlifyIdentity";
```
